spec/dor/services/client/accession_spec.rb

```diff
@@ -51,6 +53,15 @@
         it 'returns true' do
           expect(client.start(params)).to be true
         end
+
+        context 'with context' do
+          let(:body) { '{"context":{"requireOCR":true}}' }
+          let(:params) { { opening_user_name: 'dude', workflow: 'accessionWF', context: { 'requireOCR' => true } } }
+
+          it 'returns true' do
+            expect(client.start(params)).to be true
+          end
+        end
```

Comment:
the stubbed request now includes a body, if the body would be incorrectly blank, this spec will fail